tests/test_others.py

```diff
@@ -51,7 +51,7 @@ def test_create_as_with_parenthesis_around_table_name():
 
 def test_create_as_with_parenthesis_around_both():
     sql = "CREATE TABLE tab1 AS (SELECT * FROM (tab2))"
-    assert_table_lineage_equal(sql, {"tab2"}, {"tab1"})
+    assert_table_lineage_equal(sql, {"tab2"}, {"tab1"}, skip_graph_validation=True)
```

Comment:
since now we are passing the write context of target the subquery itself will not be added into graph as a write node hence the graph of sqlfluff and sqlparse won't match anymore for such cases